### PR TITLE
Doc: Prompt set

### PR DESCRIPTION
**Pull request by parniantaghipour**

### CODE REVIEW

File: docs/features/plugin/prompt-set.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Open WebUI offers a flexible interface for creating custom prompts. The prompt creation process involves the following steps:
+
+1. **Add a Title**: Assign a clear and descriptive title to your prompt, such as "Professional Email" or "Customer Query."
+2. **Define a Command**: Create a reusable command (e.g., `/email`) that allows quick insertion of the prompt content.
+3. **Set Prompt Content**: Write the core instructions, incorporating customizable placeholders.
+4. **Save the Prompt**: Store the prompt for future use, ensuring it is easily accessible.
+
+### Who Can Access and Use Prompts?
+- **Admins**: Have the ability to create and manage prompts within a shared workspace, making them available to all users in the organization.
+- **Users**: Can access these prompts but cannot directly modify the workspace content.
```
Comment (tjbck) [inline docs/features/plugin/prompt-set.md:21]: Not true with 0.4 release.

Comment (parniantaghipour) [reply docs/features/plugin/prompt-set.md:21]: Added user group file to be used here

File: docs/features/user-group.md
Comment (tjbck) [inline docs/features/user-group.md:1]: this should be in a separate PR

File: docs/features/workspace/prompt-set.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 3
+title: "📜 Prompt Presets"
+---
+
+Prompt Presets are a handy feature in Open WebUI that make working with LLM tools simpler and faster. Imagine you’ve figured out the perfect way to ask the model a specific question—let’s say for summarizing a technical document with key highlights and action points. Instead of rewriting this detailed prompt every time, you can save it as a Prompt Preset. The next time you need it, just select the preset, customize a few details if needed, and get your results instantly. This makes it easy to reuse well-crafted prompts without wasting time recreating them from scratch.
```
Comment (tjbck) [inline docs/features/workspace/prompt-set.md:6]: `LLM tools` -> `LLMs`

File: docs/features/workspace/prompt-set.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Open WebUI offers a flexible interface for creating custom prompts. The prompt creation process involves the following steps:
+
+1. **Add a Title**: Assign a clear and descriptive title to your prompt, such as "Professional Email" or "Customer Query."
+2. **Define a Command**: Create a reusable command (e.g., `/email`) that allows quick insertion of the prompt content.
+3. **Set Prompt Content**: Write the core instructions, incorporating customizable placeholders.
+4. **Save the Prompt**: Store the prompt for future use, ensuring it is easily accessible.
+
+### Who Can Access and Use Prompts?
+- **Admins**: Have the ability to create and manage prompts within a shared workspace.
```
Comment (tjbck) [inline docs/features/workspace/prompt-set.md:20]: manage all prompts

File: docs/features/workspace/prompt-set.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- **Users**: They can use the prompts added by the admin , however, they can add and edit prompts if granted the necessary [access by the admin](../user-group.md).
+
+This structured approach ensures that prompts remain standardized and accessible to relevant team members.
+
+---
+
+## How Do Variables Work in Prompts?
+
+To enhance adaptability, Open WebUI allows the use of **variables** in prompts. Variables act as placeholders that can be dynamically replaced with specific content during prompt execution. They are defined by enclosing text within double curly braces, such as `{{VARIABLE_NAME}}`.
```
Comment (tjbck) [inline docs/features/workspace/prompt-set.md:29]: rm `To enhance adaptability,`

File: docs/features/workspace/prompt-set.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### How Can Variables Be Useful?
```
Comment (tjbck) [inline docs/features/workspace/prompt-set.md:31]: Rm this section

**Comment by tjbck** [comment]
the rest lgtm
